### PR TITLE
Move commands into base image instead of Dockerfile in template

### PR DESCRIPTION
**Pull request by ewels**
I don't think that these commands need to go out into all pipelines. Simpler and less error prone if they are instead added to the base image that all other DSL1 containers are built from.

## PR checklist

 - [x] This comment contains a description of changes (with reason)
 - [ ] `CHANGELOG.md` is updated
 - [ ] If you've fixed a bug or added code that should be tested, add tests!
 - [ ] Documentation in `docs` is updated

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -2,8 +2,12 @@ FROM continuumio/miniconda3:4.8.2
 LABEL authors="phil.ewels@scilifelab.se,alexander.peltzer@qbic.uni-tuebingen.de" \
```
Comment (apeltzer) [inline Dockerfile:2]:
```suggestion
LABEL authors="phil.ewels@scilifelab.se,alexander.peltzer@boehringer-ingelheim.com" \
```

Comment (apeltzer) [reply Dockerfile:2]: Maybe we can upgrade to latest version 4.9 too ?

`FROM continuumio/miniconda3:4.9.2`

**Review by apeltzer — APPROVED** [review]
Just two things, update FROM while you're at it maybe to be using conda 4.9+ and my email is wrong ;-)

**Comment by ewels** [comment]
Done 👍🏻

**Pull request merged**